Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require "rubocops/shared/string_content"
+require "rubocops/shared/helper_functions"
+
+# TODO: move the micro shared modules above into this one?
+
+module RuboCop
+  module Cop
+    module DescHelper
+      include StringContent
+      include HelperFunctions
+
+      VALID_LOWERCASE_WORDS = %w[
+        macOS
+      ].freeze
+
+      def audit_desc(type, name, desc_call)
+        # Check if a desc is present
+        if desc_call.nil?
+          problem "#{type.to_s.capitalize} should have a desc (Description)."
+          return
+        end
+
+        desc = desc_call.first_argument
+
+        # Check the desc length. Should be >0 and <80 characters.
```

Comment:
Shouldn't this actually check the 80 character max?

Reply:
Seems like this is checked at the very bottom.

Reply:
Ah, I see. I've adjusted the comments to better reflect what is tested.